packages/styled-components/transforms/src/visitors/transpile_css_prop/transpile.rs

```diff
@@ -291,9 +332,35 @@ impl VisitMut for TranspileCssProp {
         }
     }
 
+    fn visit_mut_function(&mut self, f: &mut Function) {
+        let mut new_params: AHashSet<JsWord> = AHashSet::default();
+        for param in &f.params {
+            new_params.extend(collect_top_level_decls(param));
+        }
+        self.bindings.push(new_params);
+        self.bindings.push(collect_top_level_decls(f));
+        f.body.visit_mut_with(self);
+        self.bindings.pop().unwrap();
+        self.bindings.pop().unwrap();
+    }
+
+    fn visit_mut_arrow_expr(&mut self, e: &mut ArrowExpr) {
```

Comment:
Same here.
(I think I made mistke while reviewing the PR on `next-swc`, sorry for it)